src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -227,6 +230,15 @@
         - name: --disable-pod-security-policy
           type: bool
           short-summary: (PREVIEW) Disable pod security policy.
+        - name: --enable-acr
+          type: bool
+          short-summary: (PREVIEW) Grant the 'acrpull' role assignment for the ACR set by --acr-name or --acr-resource-id.
+        - name: --acr-name
+          type: string
+          short-summary: (PREVIEW) The name of the ACR in AKS resource group.
+        - name: --acr-resource-id
```

Comment:
the convention is to combine `acr_name` and `azr_resource_id` into `acr`. It is very easy to detect if it is not a resource id then it is a name. Check out how VM is doing [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/vm/_validators.py#L45)

Reply:
agree, It will be concise.

Reply:
The user interface design is from PM side. I think using different options is also more clear

cc @sauryadas

Reply:
works for me .. this is concise and we of course should follow conventions. I did not know this existed.
@feiskyer in the CLI help , the help text will suggest that --acr can take either acr-name or acr-id .. please update the description

Reply:
Thanks, would merge these two options into `--acr`.